From 1081c82f1b12a48521c90d47e61ef623237d240f Mon Sep 17 00:00:00 2001
From: Mel Gorman <mgorman@suse.de>
Date: Wed, 18 Jun 2014 00:23:05 +0100
Subject: [PATCH] cfq: Increase default value of target_latency

References: VM/FS Performance
Patch-mainline: No (Expected 3.17)

The existing latency looks like a performance regression for some IO tests
on single-disk systems. While this can be easily described as a tuning
parameter it is one that is relatively difficult to detect. Increase the
default on the assumption that people with access to expensive fast storage
also know how to tune their IO scheduler.

Signed-off-by: Mel Gorman <mgorman@suse.de>
---
 block/cfq-iosched.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/block/cfq-iosched.c b/block/cfq-iosched.c
index 434944c..a2e5031 100644
--- a/block/cfq-iosched.c
+++ b/block/cfq-iosched.c
@@ -32,7 +32,7 @@ static int cfq_slice_async = HZ / 25;
 static const int cfq_slice_async_rq = 2;
 static int cfq_slice_idle = HZ / 125;
 static int cfq_group_idle = HZ / 125;
-static const int cfq_target_latency = HZ * 3/10; /* 300 ms */
+static const int cfq_target_latency = HZ * 6/10; /* 300 ms */
 static const int cfq_hist_divisor = 4;
 
 /*
